### ISSUE TITLE
Is this real-time behavior recognition?

### ISSUE THREAD

**Issue by JoshaPan** [kenshohara/3D-ResNets-PyTorch#204]
I hope you can reply to this question?
thnaks

**Comment by hyq123-cmd** [comment]
I don't think so ! Beacuse the weight of the resnet-34 is already 400M that is difficult to achieve real-time.

**Comment by guilhermesurek** [comment]
I run a resnet-50 on the webcam and got 8 fps on a GTX 1660 Ti.
